cpi-api-v1.html.md.erb

```diff
@@ -115,15 +116,27 @@ Make sure to properly delete created resources if VM cannot be successfully crea
 	  	"private2": {
 			"type": "dynamic",
 			"cloud_properties": { "net_id": "subnet-e12364" }
-		}
+		},
 		"public": {
 			"type": "vip",
 			"ip": "173.101.112.104",
 			"cloud_properties": {}
 		}
 	},
 	[ "vol-3475945" ],
-	{}
+	{
+		"bosh": {
+			"group": "bosh_director_name-deployment_name-instance_group_name",
+			"groups": [
+				"bosh_director_name",
```

Comment:
let's not add actual names since they should be opaque. otherwise it's a bit misleading in terms of promising some format.